ipython_profiles/profile_s6bm/startup/03-plans.py

```diff
@@ -140,6 +141,7 @@ def scan_closure():
                 yield from bps.mv(preci, ang)
                 yield from bps.trigger_and_read([det])
         elif config['tomo']['type'].lower() == 'fly':
+            yield from bps.mv(det.proc1.num_filter, 1)
```

Comment:
Would any sequencing be lost if these three lines were combined into a single `yield from bps.mv(...)`?

Reply:
Based on previous testings, the sequence of setting each parameters matters and we are in the process of catching all the hidden _tribal knowledge_ through trial and error.  Hence the verbose syntax used in the current version.